proto.bat

```diff
@@ -1,12 +1,12 @@
-echo "Creating proto files"
-
-for /f %%i in ('dir /a:d /s /b api') do %CD%\replay_analysis\generated\protoc.exe --proto_path=%CD% --python_out=%CD%\replay_analysis\generated %%i\*.proto
-%CD%\replay_analysis\generated\protoc.exe --proto_path=%CD% --python_out=%CD%\replay_analysis\generated %CD%\api\*.proto
-
-echo "Proto files are created"
-
-echo "Fixing protobuf relative imports dumbness"
-
-python ./import_fixer.py
-
-echo "Protobuf imports are now fixed"
+echo "Creating proto files"
+
+for /f %%i in ('dir /a:d /s /b api') do "%CD%"\replay_analysis\generated\protoc.exe --proto_path="%CD%" --python_out="%CD%"\replay_analysis\generated %%i\*.proto
+"%CD%"\replay_analysis\generated\protoc.exe --proto_path="%CD%" --python_out="%CD%"\replay_analysis\generated "%CD%"\api\*.proto
```

Comment:
did you double check to make sure this will not break anyone else.
or got this working for you

Reply:
no it's not working for me. I didn't double check with anyone else as this shouldn't break anything. I just added ""s around the %CD%s, and it changed the error for me. (from being about space in directory to being about the *.